docs/src/@primer/gatsby-theme-doctocat/nav.yml

```
@@ -1,11 +1,15 @@
-- title: Getting started
-  url: /getting-started
+- title: Introduction
+  url: /
```

The url field is ignored on the parent item so we can delete it from:

```suggestion
- title: Introduction
```